reactnative/RNPlayground/app/screens/AboutScreen.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FC } from "react"
+import { observer } from "mobx-react-lite"
+import { StackScreenProps } from "@react-navigation/stack"
+import { AppStackScreenProps } from "../navigators"
+import { Image, ImageStyle, TextStyle, View, ViewStyle } from "react-native"
+import {
+  Text, Screen
+} from "../components"
+import { isRTL } from "../i18n"
+import { colors, spacing } from "../theme"
+import { useSafeAreaInsetsStyle } from "../utils/useSafeAreaInsetsStyle"
+
+export const AboutScreen: FC<StackScreenProps<AppStackScreenProps, "About">> = observer(function AboutScreen() {
+    const $bottomContainerInsets = useSafeAreaInsetsStyle(["bottom"])
+
+    return (
+      <View style={$container}>
+        <View style={$topContainer}>
+          <Text
+            testID="welcome-heading"
+            style={$welcomeHeading}
+            tx="aboutScreen.readyForLaunch"
+            preset="heading"
+          />
+          <Text tx="aboutScreen.postscript" preset="subheading" />
+        </View>
+
+        <View style={[$bottomContainer, $bottomContainerInsets]}>
+          <Text tx="aboutScreen.exciting" size="md" />
+        </View>
+      </View>
+    )
+})
+
+const $container: ViewStyle = {
+  flex: 1,
+  backgroundColor: colors.background,
+}
+
+const $topContainer: ViewStyle = {
+  flexShrink: 1,
+  flexGrow: 1,
+  flexBasis: "57%",
+  justifyContent: "center",
+  paddingHorizontal: spacing.large,
+}
+
+const $bottomContainer: ViewStyle = {
+  flexShrink: 1,
+  flexGrow: 0,
+  flexBasis: "43%",
+  backgroundColor: colors.palette.neutral100,
+  borderTopLeftRadius: 16,
```

Comment:
border radius is something that you will probably want to keep track of, design wise, to apply the same on other elements. I encourage you to detect duplications of this `16` or keep track of them to make sure there is a single place where to have them for reference. 

If there are no duplications for the time being it's fine, no need to extract, just keep track of them